package/MDAnalysis/coordinates/LAMMPS.py

```diff
@@ -458,19 +458,53 @@ class DumpReader(base.ReaderBase):
     "unwrapped" (xu,yu,zu) and "scaled_unwrapped" (xsu,ysu,zsu) coordinate
     conventions (see https://docs.lammps.org/dump.html for more details).
     If `lammps_coordinate_convention='auto'` (default),
-    one will be guessed. Guessing checks whether the coordinates fit each convention in the order "unscaled",
-    "scaled", "unwrapped", "scaled_unwrapped" and whichever set of coordinates
-    is detected first will be used. If coordinates are given in the scaled
-    coordinate convention (xs,ys,zs) or scaled unwrapped coordinate convention
-    (xsu,ysu,zsu) they will automatically be converted from their
-    scaled/fractional representation to their real values.
+    one will be guessed. Guessing checks whether the coordinates fit each
+    convention in the order "unscaled", "scaled", "unwrapped",
+    "scaled_unwrapped" and whichever set of coordinates is detected first will
+    be used. If coordinates are given in the scaled coordinate convention
+    (xs,ys,zs) or scaled unwrapped coordinate convention (xsu,ysu,zsu) they
+    will automatically be converted from their scaled/fractional representation
+    to their real values.
 
     Supports both orthogonal and triclinic simulation box dimensions (for more
     details see https://docs.lammps.org/Howto_triclinic.html). In either case,
     MDAnalysis will always use ``(*A*, *B*, *C*, *alpha*, *beta*, *gamma*)``
     to represent the unit cell. Lengths *A*, *B*, *C* are in the MDAnalysis
     length unit (Å), and angles are in degrees.
 
+    By using the keyword `additional_columns`, you can specify arbitrary data
+    to be read alongside the coordinates. If specified, the keyword expects a
+    list of the names of the columns that you want to have read. The results
+    of the parsing are saved to the time step `data` dictionary alongside the
+    name of the data column. For instance, if you have time-dependent charges
+    saved in a LAMMPS dump such as
+
+    .. code-block:: python
+
+        ITEM: ATOMS id x y z q l
+        1 2.84 8.17 -25 0.00258855 1.1
+        2 7.1 8.17 -25 6.91952e-05 1.2
+
+    Then you may parse the additional columns `q` and `l` via.
+
+    .. code-block:: python
+
+        u = mda.Universe('path_to_data', 'path_to_lammpsdump',
```

Comment:
instead of explanatory filenames, use real-looking ones such as lammps.data and lammps.dump (whatever is typical but with generic names) and if you need to explain something, do it in text, referring to the filenames. In this case, no explanation is necessary as anyone reading these docs knows how to create Universe.

Reply:
Address this please, just use something plausible sounding.

Reply:
understood, makes sense